flarestack/data/icecube/ic_season.py

```diff
@@ -51,27 +51,26 @@
 
     icecube_dataset_dir = Path(flarestack_dataset_dir)
 
-    ref_7yr_path: Path = icecube_dataset_dir / mirror_7yr_dirname
-    if ref_7yr_path.is_dir():
-        ref_dir_7yr: Optional[Path] = ref_7yr_path
-    else:
-        logger.warning(f"No 7yr sensitivity directory found at {ref_7yr_path}")
+    if host_server == "DESY":
+        ref_dir_7yr = icecube_dataset_dir / "ref_sensitivity" / mirror_7yr_dirname
+        ref_10yr = icecube_dataset_dir / "ref_sensitivity" / ref_10yr_filename
+
+    elif host_server == "WIPAC":
+        ref_dir_7yr = WIPAC_7yr_dir
+        ref_10yr = WIPAC_10yr_dir / ref_10yr_filename
```

Comment:
This is now lacking a default definition for when `flarestack` is not running either at DESY or WIPAC.